weather.json

```diff
@@ -0,0 +1 @@
:5.1,"precipitation_amount_min":0.0,"probability_of_precipitation":43.1}}}},{"time":"2023-07-05T12:00:00Z","data":{"instant":{"details":{"air_pressure_at_sea_level":1004.1,"air_temperature":16.8,"air_temperature_percentile_10":13.9,"air_temperature_percentile_90":20.2,"cloud_area_fraction":99.6,"cloud_area_fraction_high":55.1,"cloud_area_fraction_low":79.3,"cloud_area_fraction_medium":63.7,"dew_point_temperature":13.9,"relative_humidity":82.4,"wind_from_direction":218.9,"wind_speed":2.6,"wind_speed_percentile_10":1.4,"wind_speed_percentile_90":3.5}},"next_12_hours":{"summary":{"symbol_code":"cloudy","symbol_confidence":"uncertain"},"details":{"probability_of_precipitation":43.1}},"next_6_hours":{"summary":{"symbol_code":"cloudy"},"details":{"air_temperature_max":18.3,"air_temperature_min":15.9,"precipitation_amount":0.0,"precipitation_amount_max":0.9,"precipitation_amount_min":0.0,"probability_of_precipitation":29.4}}}},{"time":"2023-07-05T18:00:00Z","data":{"instant":{"details":{"air_pressure_at_sea_level":1004.1,"air_temperature":15.9,"air_temperature_percentile_10":13.8,"air_temperature_percentile_90":18.7,"cloud_area_fraction":96.5,"cloud_area_fraction_high":56.6,"cloud_area_fraction_low":29.3,"cloud_area_fraction_medium":55.9,"dew_point_temperature":13.1,"relative_humidity":83.1,"wind_from_direction":257.1,"wind_speed":2.2,"wind_speed_percentile_10":1.2,"wind_speed_percentile_90":3.4}},"next_12_hours":{"summary":{"symbol_code":"cloudy","symbol_confidence":"uncertain"},"details":{"probability_of_precipitation":47.1}},"next_6_hours":{"summary":{"symbol_code":"cloudy"},"details":{"air_temperature_max":15.9,"air_temperature_min":12.6,"precipitation_amount":0.0,"precipitation_amount_max":1.3,"precipitation_amount_min":0.0,"probability_of_precipitation":35.3}}}},{"time":"2023-07-06T00:00:00Z","data":{"instant":{"details":{"air_pressure_at_sea_level":1005.6,"air_temperature":12.6,"air_temperature_percentile_10":10.5,"air_temperature_percentile_90":16.2,"cloud_area_fraction":96.1,"cloud_area_fraction_high":51.6,"cloud_area_fraction_low":50.0,"cloud_area_fraction_medium":39.1,"dew_point_temperature":11.8,"relative_humidity":94.9,"wind_from_direction":142.2,"wind_speed":1.8,"wind_speed_percentile_10":1.0,"wind_speed_percentile_90":4.0}},"next_6_hours":{"summary":{"symbol_code":"cloudy"},"details":{"air_temperature_max":14.6,"air_temperature_min":12.2,"precipitation_amount":0.0,"precipitation_amount_max":0.8,"precipitation_amount_min":0.0,"probability_of_precipitation":29.4}}}},{"time":"2023-07-06T06:00:00Z","data":{"instant":{"details":{"air_pressure_at_sea_level":1006.0,"air_temperature":14.4,"air_temperature_percentile_10":11.7,"air_temperature_percentile_90":16.0,"cloud_area_fraction":94.5,"cloud_area_fraction_high":14.8,"cloud_area_fraction_low":52.3,"cloud_area_fraction_medium":6.2,"dew_point_temperature":12.3,"relative_humidity":86.7,"wind_from_direction":157.2,"wind_speed":2.1,"wind_speed_percentile_10":0.8,"wind_speed_percentile_90":4.1}}}}]}}
```

Comment:
Accidental add?

Reply:
Doh, first time trying git in vscode 🤦 

Let me fix it in emacs..

Reply:
It autoformatted too 😭 ..